src/state/state-service.ts

```diff
@@ -273,30 +274,37 @@ export default class StateService {
             throw new Error(`Can't located artefact ${artefactId} in edition ${editionId}`);
         }
 
-        const imagedObject = this._state.imagedObjects.find(artefact.imagedObjectId);
-        if (!imagedObject) {
-            console.error(`Can't locate imaged object ${artefact.imagedObjectId} for artefact ${artefact.id}`);
-            throw new Error(`Can't locate imaged object ${artefact.imagedObjectId} for artefact ${artefact.id}`);
-        }
+        const promises: Array<Promise<any>> = [];
```

Comment:
Looks like the type should be whatever `this.imageManifest()` returns, or did I miss reason why we need to use the type `any`?

Reply:
We did not create a typescript type for the image manifest, so imageManifest is an any. We have an interface for the tile info, I guess we should create an interface for the parts of the manifests we use. I don't want this to delay progress in more crucial issues.

Reply:
That is fine.